apps/institution-service/package.json

```diff
@@ -20,12 +20,14 @@
     "test:db:seed": "npx sequelize-cli db:seed:all --env test",
     "test:db:reset": "npx sequelize-cli db:drop --env test && npm run test:db:setup",
     "dev": "nodemon ./src/app.ts",
+    "start": "npx ts-node --transpile-only ./src/app.ts",
```

Comment:
Why do we need a start command and a procfile? Is the start command just to test that the procfile command will work?

Reply:
It's actually pointless in the package json, so I'll remove it. We need to use Procfile because the heroku apps allow an env variable to point to a specific procfile so that we can deploy different services on the same repo.